common/metrics/defs.go

```diff
@@ -789,6 +789,7 @@ const (
 	StandbyTransferTaskQueueLatency
 	StandbyTimerTaskQueueLatency
 	CompleteTaskFailedCounter
+	WorkflowConcurrentExecutionCounter
```

Comment:
Can we define this as a PersistenceErr Counter similar to 'PersistenceErrConditionFailedCounter'